Decred Wallet/Features/Security/PasswordSetupViewController.swift

```diff
@@ -9,19 +9,35 @@
 import UIKit
 
 class PasswordSetupViewController: SecurityBaseViewController, UITextFieldDelegate {
-    @IBOutlet weak var headerText: UILabel!
-    @IBOutlet weak var tfPassword: UITextField!
-    @IBOutlet weak var tfConfirmPassword: UITextField!
-    @IBOutlet weak var lbMatchIndicator: UILabel!
+    // Input fields
+    @IBOutlet weak var tfPassword: FloatingLabelTextInput!
+    @IBOutlet weak var tfConfirmPassword: FloatingLabelTextInput!
+    
     @IBOutlet weak var pbPasswordStrength: UIProgressView!
     
+    @IBOutlet weak var passwordCount: UILabel!
+    @IBOutlet weak var confirmCount: UILabel!
+    
+    // Buttons
+    @IBOutlet weak var cancelBtn: UIButton!
+    @IBOutlet weak var createBtn: UIButton!
+    
+    // Error Labels
+    @IBOutlet weak var passwordErr: UILabel!
+    @IBOutlet weak var confirmErr: UILabel!
+    
```

Comment:
A single empty line space before and after the following 2 declarations is sufficient.